### PR TITLE
fix #79681 4anime.to

### LINKED ISSUE #79681: 4anime.to
### Issue URL (Annoyance)
 [https://4anime.to/one-piece-episode-267?id=2097](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2F4anime.to%2Fone-piece-episode-267%3Fid%3D2097)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.34
Browser: | Chrome
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion<br/><br/><b>Language-specific:</b><br/> AdGuard Japanese

### PR DESCRIPTION
**Pull request by wout-codes**
#79681

### CODE REVIEW

**Comment by Yuki2718** [comment]
On my end already moved to `||gaijiglo.net^`. In any case add `#%#//scriptlet("abort-current-inline-script", "Math", "/emZnbG9hZGVk|break;case \$\./")` too.

**Review by Yuki2718 — APPROVED** [review]

**Comment by wout-codes** [comment]
Yeah, I have known that. But the regex stops the adservers (bootstrap).

File: EnglishFilter/sections/general_extensions.txt [hunk 1]
```diff
@@ -163,7 +163,7 @@ nyafilmer1.com,eririo.club,upload-4ever.com,dlpsgame.*,allkindonline.com,123free
...
 ! `#%#//scriptlet("abort-current-inline-script", "Math", "/emZnbG9hZGVk|break;case \$\./")`
-streamtape.com#%#//scriptlet("abort-current-inline-script", "Math", "/emZnbG9hZGVk|break;case \$\./")
+4anime.to,streamtape.com#%#//scriptlet("abort-current-inline-script", "Math", "/emZnbG9hZGVk|break;case \$\./")
```
Comment (Alex-302) [inline EnglishFilter/sections/general_extensions.txt:166]: why new rule? Is `#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "delete window")` not working?

Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:166]: It's working, but @Yuki2718 Suggested it. I agree on both. @Yuki2718 What do you suggest?

Comment (Yuki2718) [reply EnglishFilter/sections/general_extensions.txt:166]: Sorry, it's my bad.

Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:166]: Sure.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**